routes/index.js

```diff
@@ -9,6 +9,7 @@ router.get('/', (req, res, next) => {
 
 // pipe all other requests through the route modules
 router.use(require('./authRoute'));
+router.use(require('./productDetailRoute'));
```

Comment:
As you go forward, you might want to be more inclusive about this, but it's not absolutely necessary. But won't productDetailRoute only have one route in it..?

Reply:
....nevermind. It probably will have edit/delete options too. Still, you may end up wanting to group your product routes together. This is up to you, however.

Reply:
Hey @Rian501, thanks for the great suggestion! However, we have decided to stick with our Product Detail Ctrl at this time. Let me know if you have any questions!